src/app/personal-details/page.tsx

```diff
@@ -14,12 +14,13 @@ import GradientButton from "../_components/GradientButton";
 
 const PersonalDetailsScreen: React.FC = () => {
 	const [name, setName] = useState("");
-	const [username, setUsername] = useState("");
```

Comment:
are we not using a username of are we just going to use the names provided by google during sign in.

Reply:
Hey! Do you mind clarifying your comment?

Reply:
I'll go ahead and solve this comment as we talked about it in person.